README.md

```diff
@@ -140,6 +140,7 @@ List of useful, silly and [awesome](#awesome-) lists curated on GitHub. Contribu
 * [datasciencemasters](https://github.com/datasciencemasters/go) – The Curriculum for learning Data Science, Open Source and at your fingertips.
   - http://datasciencemasters.org/
 * [DataSciencePython](https://github.com/ujjwalkarn/DataSciencePython) – Python tutorials for Data Science, NLP and Machine Learning
+* [Decision Tree Papers](https://github.com/benedekrozemberczki/awesome-decision-tree-papers) – Decision Tree Research Papers
```

Comment:
Hi Benedek, thank you for your PR! Could you please rename the link to match repo name and move it to the `awesome-*` section? Just like your other PRs.